src/openstudio_lib/library/geometry_preview.html

```diff
@@ -1412,22 +1443,58 @@
   });
   f1.open();
 
-  var f2 = gui.addFolder('Potential Adjacency Issues');
-  f2.add(settings, 'findAdjacencyIssues').name('Enable').onChange(value => {
-      removeSelection();
-      setLocalStorage('findAdjacencyThreshold', value);
-      updateAdjacencyIssues();
-      update(value);
-  });
-  f2.add(settings, 'findAdjacencyThreshold', 0.1, 10, 0.5).name('Threshold').onFinishChange(function (value) {
-      if (settings.findAdjacencyIssues) {
+  var diagnostics = gui.addFolder("Diagnostic Tools")
```

Comment:
Consider moving all "Diagnostic Tools" under check for `includeGeometryDiagnostics`?

Reply:
So, the geometry diagnostics checkbox affect the way the ThreeJS translation is done (include those, with computational cost, or do not). The adjacency issue and co were already there and aren't affected.

If we think it's better to hide the entire menu we can do that though.

Reply:
My only concern is that people might click "Geometry Diagnostics" and then not be able to find the new menu items.  Another idea would be to rename the checkbox "Extra Geometry Diagnostics" so people know that new items will be added under "Diagnostic Tools".  It's fine just how it is, only make any changes if you think it's an improvement.

Reply:
Hiding the entire "Diagnostics Tools" folder when the checkbox is unclicked. adbd297e6cdefc97e3f21633b1fb9c669a16d2d7

Reply:
Thanks Julien, I think this is nicer